mkdocs/docs/developer/frontend.md

```diff
@@ -75,6 +77,47 @@ The files that should be manipulated by developers are housed within the `src/`
 - **frontend/src/:** houses all the files for developers to manipulate. The files here are read by webpack before being bundled into the `static/` and `templates/` directories.
   - **assets/:** this is where we store all of our miscellaneous files, such as .jpegs, .svgs, .gifs, etc.
   - **componenents/:** this is where we store the files that generate our components, such as buttons and cards. To learn more about this in-depth, read the [components](#components-directory) section of this guide.
+  - **context/:** contains the logic and data management utilities related to context providers and consumers. Contexts are used for managing global state within our application, providing a way to pass data through the component tree without having to pass props manually at every level.
+    - **COP (Community of Practice) JSON Structure:** The COP data represents different communities of practice within our organization, each consisting of various roles and descriptions. Below is the JSON structure of the COP data for QualifierPageRoles.tsx:
+    ```
+    {
+      COPs: {
```

Comment:
Can we change the structure to not have whitespaces ? Instead of "Back End Developer" , we should use "Back_End_Developer" ... etc. plz

Reply:
Hey @JimmyJuarez10 , Thanks for your feedback! I can change the formatting in this JSON structure. 
I wanted to confirm with you: 
Should I also change the format of strings in frontend/src/api_data/copData.ts? 
If so, that means I should replace all underscores with spaces on line 145 in frontend/src/pages/QualifierPage/QualifierPageRoles.tsx, correct? Currently, the strings are used as text values for each Chip.

Reply:
mmm to clarify , right now we are reading the value off the chip and storing that in the json ?